### DISCUSSION TITLE
Never used Github but for some reason lost money from my bank card<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by RulonOG** [community/community#120494] [Discussions]
### Select Topic Area

Question

### Body

Hello every one! 

I don't know how and who do I contact about my problem. Never been using Github but for some reason lost money from my bank card. I blocked furture payments and blocked my bank card after I found out this. How do I get my money back?

I got statement for this payment but don't know how to add it here.

**Comment by ettaboyle** [comment]
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
